x-pack/platform/plugins/shared/fleet/server/tasks/sync_integrations/custom_assets.ts

```diff
@@ -114,13 +118,54 @@ export const getCustomAssets = async (
     []
   );
 
-  const updatedAssets = [...customAssetsComponentTemplates, ...customAssetsIngestPipelines];
+  const customPipelineFromVars = await getPipelinesFromVars(esClient, soClient, abortController);
+
+  const updatedAssets = [
+    ...customAssetsComponentTemplates,
+    ...customAssetsIngestPipelines,
+    ...customPipelineFromVars,
+  ];
 
   const deletedAssets = updateDeletedAssets(previousSyncIntegrationsData, updatedAssets);
 
   return [...updatedAssets, ...deletedAssets];
 };
 
+export async function getPipelinesFromVars(
+  esClient: ElasticsearchClient,
+  soClient: SavedObjectsClient,
+  abortController: AbortController
+): Promise<CustomAssetsData[]> {
+  const packagePolicies = await packagePolicyService.list(soClient, {
```

Comment:
we can't filter on input packages, there are some packages that have integration type (e.g. filestream)

Reply:
I think we will have to search across all space here for this work with space awareness enabled like [here](https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/shared/fleet/server/services/output.ts#L174-L178)

I am wondering if we should have an internal package policy service with a specific method for this kind of usage